traits/trait_handlers.py

```diff
@@ -2537,7 +2537,7 @@ def __setitem__(self, key, value):
                     # In this case, we return a TraitListEvent with an
                     # index=key.start and the removed and added lists as they
                     # are.
```

Comment:
do we need to update the FIXME comment about this change?

Reply:
That's about a different thing; nothing to do with the use of `None`. Admittedly it's not strictly true that we're using exactly `index=key.start` any more, but I think it's close enough that the comment shouldn't be confusing.

Reply:
👍